LUMO_toolbox/DOTHUB_LUMOdataQualityCheck.m

```diff
@@ -58,9 +64,91 @@ function DOTHUB_dataQualityCheck(nirsFileName,printFigFlag)
 dists = DOTHUB_getSDdists(SDtmp);
 
 % MOTION CHECK  #######################################################
-% - this script does not focus on motion content, but providing
+%timeSelectFlag allows the user to select a clean period (motion free) for analysis. 
+%A subset of channels are plotted to decrease computer load (this can
+%always be changed to plot all channels).
+if (timeSelectFlag==1) 
+    my_ans = 'n';
+    while my_ans == 'n';
+    fig1 = figure; set(fig1, 'units', 'normalized', 'outerposition', [0 0 0.8 0.8], 'color', [1 1 1])
+    plot(d(:,1500:2000)); xlim([0 size(d,1)]); ylim([0 1.5]); title('Select segment to include (start-end pair) - Press enter when finished', 'fontsize', 20)
```

Comment:
Don't hard-code indices - this is something you should always avoid when coding.  What would happen if we were using only 2 tiles?

Reply:
If you want to reduce the number of channels plotted, try randomly selecting 500 channels, or plotting all channels if number of channels <= 500

Reply:
I did this so now it selects 500 channels randomly